src/NubankClient/Model/Savings/Saving.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using NubankClient.Converters;
+using System;
+using System.Diagnostics;
+
+namespace NubankClient.Model
```

Comment:
@natenho I think that the namespace should represent the file path, I would use NubankClient.Model.Savings.

Reply:
You are right! My bad! Fixed